openwisp_monitoring/monitoring/static/monitoring/js/chart.js

```diff
@@ -97,7 +97,8 @@
                 options.marker = {
                     cmax: config.max,
                     cmin: config.min,
-                    colorbar: {title: config.label},
+                    colorbar:
+                        config.hide_colorbar === true ? null : {title: config.label},
```

Comment:
I introduced this new parameter, because I think it doesn't make much sense to keep the color bar in the `access technology` chart. Tell me if this is not desirable.